src/generator/text-to-state-machine.js

```diff
@@ -18,7 +18,23 @@ export default (code) => {
     const {results} = parser;
     const [result] = results;
 
-    const {states} = result;
+    const {states, timers} = result;
+
+    // Convert all integer states to int
+    states.forEach(({state}, index) => {
+        if(!isNaN(state)) {
+            states[index].state = parseInt(state);
+        }
+    });
+
+    // Convert all integer transitions to int
+    states.forEach(({transitions}, stateIndex) => {
+        transitions.forEach(({goto}, transitionIndex) => {
+            if(!isNaN(goto)) {
+                states[stateIndex].transitions[transitionIndex].goto = parseInt(goto);
```

Comment:
Could this step be removed by altering the tokenizer? I noticed that the unary `+` was removed.

Reply:
That would be the best yeah, I didn't knew how to do that, so I made a loop for this, but if the tokenizer can do it ...

Reply:
Fair. I think all you'd have to do is move the `isNaN` check into `grammar.ne`

Reply:
I just pushed a commit for this, it also remove the conversion for states labels too, so text-to-state-machine.js is now untouched by this PR